include/nektar_interface/particle_cell_mapping/generated_linear/linear_newton_implementation.hpp

```diff
@@ -0,0 +1,10 @@
+#ifndef __GENERATED_LINEAR_NEWTON_IMPLEMENTATIONS_H__
```

Comment:
No triangles?

Reply:
Linear sided triangles are always regular (I think) and hence covered by the regular mapper.